scripts/run_benchmarks.sh

```
@@ -0,0 +1,169 @@
+    echo "以下のコマンドでPostgreSQLを起動してください:"
+    echo "  brew services start postgresql"
+    echo "  # または"
+    echo "  docker-compose up -d postgres"
+fi
+
+# ベンチマーク用データベースが存在するか確認
+DB_EXISTS=$(psql "$BENCHMARK_DB_URL" -c "SELECT 1" 2>/dev/null | grep -c "1" || echo "0")
+if [ "$DB_EXISTS" -eq "0" ]; then
+    echo -e "${YELLOW}ベンチマーク用データベースが見つかりません。作成しています...${NC}"
+    DB_NAME=$(echo "$BENCHMARK_DB_URL" | sed 's/.*\///')
+    BASE_URL=$(echo "$BENCHMARK_DB_URL" | sed 's/\/[^/]*$//')
```

Shellcheck (0.10.0)

[warning] BASE_URL appears unused. Verify use (or export if used externally).

(SC2034)